src/CheckGroup/index.js

```diff
@@ -6,7 +6,7 @@ const CheckGroup = ({ options, value, type, onChange }) => {
   return (
     <div role="radiogroup" className="ola_checkGroup" onChange={ e => onChange(e) }>
       { options && options.map( (option, idx) => (
-        <Check key={idx} name="test" type={type} value={option.value} checked={value && value == option.value}>
+        <Check key={idx} name={name || 'checkGroup'} type={type} value={option.value} checked={value && value == option.value}>
```

Comment:
- Not sure if it's good idea to give a default name. Maybe make this option mandatory?
- What is the purpose of `key` property? It's not defined in `Check` component

Reply:
- No is the better idea, but it´s a not breaker change. 
- Key is required in maps by react: https://reactjs.org/docs/lists-and-keys.html

Reply:
The problem with a default name is that is shared by all `Check` in the page. If there're two `CheckGroup` at the same time, there's a conflict between them. So, if it's not mandatory, I'm inclined to make it optional, but without assign a default name.